dali/base/dafdesc.cpp

```diff
@@ -1323,6 +1324,9 @@ class CFileDescriptor:  public CFileDescriptorBase, implements ISuperFileDescrip
         attr.setown(createPTree(mb));
         if (!attr)
             attr.setown(createPTree("Attr")); // doubt can happen
+        history.setown(createPTree(mb));
```

Comment:
Need to be careful about backward and forward compatibility when altering serialization.